bin/supply

```diff
@@ -16,26 +16,99 @@ BUILDPACK_DIR=$(dirname $BIN_DIR)
 
 echo "[cyberark-conjur-buildpack]: supplying"
 
-if [ ! -f $BUILD_DIR/secrets.yml ]; then
-  echo "No secrets.yml file found in $BUILD_DIR."
+
+echo "[cyberark-conjur-buildpack]: looking for secrets.yml"
+if [ -f $BUILD_DIR/secrets.yml ]; then
+  echo " - secrets.yml file found in $BUILD_DIR"
+elif  [ -f $BUILD_DIR/BOOT-INF/classes/secrets.yml ]; then
+  echo " - secrets.yml file found in $BUILD_DIR/BOOT-INF/classes/secrets.yml copy to $BUILD_DIR/secrets.yml"
+  cp $BUILD_DIR/BOOT-INF/classes/secrets.yml $BUILD_DIR/secrets.yml
+else
+  echo " - Unable to find a secrets.yml... exit"
   exit 1
 fi
 
+echo "[cyberark-conjur-buildpack]: looking for cyberark-conjur in VCAP_SERVICES"
 if [[ false = $(echo $VCAP_SERVICES | jq 'has("cyberark-conjur")') ]]; then
   echo "No credentials for cyberark-conjur service found in VCAP_SERVICES."
```

Comment:
Let's go ahead and bring the formatting for this message in line with your others.

```suggestion
  echo " - No credentials for cyberark-conjur service found in VCAP_SERVICES... exit"
```